src/gpu-compute/GPU.py

```diff
@@ -150,6 +158,11 @@ class ComputeUnit(ClockedObject):
     dpbypass_pipe_length = Param.Int(
         4, "vector ALU Double Precision bypass latency"
     )
+
+    rfc_pipe_length = Param.Int(
+        2, "vector ALU Double Precision bypass latency"
```

Comment:
I'm not sure the description is right here -- seems a copy-paste error from above.